Code/IO/include/sitkImageViewer.h

```diff
@@ -0,0 +1,205 @@
+/*=========================================================================
+*
+*  Copyright Insight Software Consortium
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*         http://www.apache.org/licenses/LICENSE-2.0.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+*=========================================================================*/
+#ifndef sitkImageViewer_h
+#define sitkImageViewer_h
+
+#include "sitkImage.h"
+#include "sitkIO.h"
+#include "sitkProcessObject.h"
+
+namespace itk
+{
+namespace simple
+{
+
+/** \class ImageViewer
+  * \brief Display an image in an external viewer (Fiji by default)
+  *
+  * The ImageViewer class displays an image with an external image display
+  * application.  By default the class will search for a Fiji ( https://fiji.sc )
+  * executable.  The image is written out to a temporary file and then passed
+  * to the application.
+  *
+  * When SimpleITK is first invoked the following environment variables are
+  * queried to set up the external viewer:
+  *
+  * SITK_SHOW_EXTENSION:  file format extension of the temporary image file.
+  * The default is '.mha', the [MetaIO](https://itk.org/Wiki/ITK/MetaIO)
+  * file format.
+  *
+  * SITK_SHOW_COMMAND:  The user can specify an application other than Fiji
+  * to view images.
+  *
+  * These environment variables are only checked at SimpleITK's launch.
+  */
+class SITKIO_EXPORT ImageViewer
+{
+
+public:
+  ImageViewer();
+
```

Comment:
We should also have a `virtual std::String GetName() const` method like other SimpleITK objects.